bitnami/redmine/README.md

```diff
@@ -58,132 +58,223 @@ helm install my-release bitnami/redmine --set databaseType=postgresql
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `persistence.enabled`                | Enable persistence using PVC                                                                                                                        | `true`                                                  |
+| `persistence.existingClaim`          | The name of an existing PVC                                                                                                                         | `nil`                                                   |
+| `persistence.storageClass`           | PVC Storage Class                                                                                                                                   | `nil` (uses alpha storage class annotation)             |
+| `persistence.accessMode`             | PVC Access Mode                                                                                                                                     | `ReadWriteOnce`                                         |
+| `persistence.size`                   | PVC Storage Request                                                                                                                                 | `8Gi`                                                   |
+
+### Pod Disruption parameters
+
+| Parameter                            | Description                                                                                                                                         | Default                                                 |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `podDisruptionBudget.enabled`        | Pod Disruption Budget toggle                                                                                                                        | `false`                                                 |
+| `podDisruptionBudget.minAvailable`   | Minimum available pods                                                                                                                              | `nil`                                                   |
+| `podDisruptionBudget.maxUnavailable` | Maximum unavailable pods                                                                                                                            | `nil`                                                   |
+
+### Custom Certificates parameters
```

Comment:
It looks like we are missing volumePermissions parameters

Reply:
Added, thanks!